### PR TITLE
Add a spec for registration form.

### LINKED ISSUE #280: Remove these fields from schema and controller
Looks like these were removed from the patients view, but there wasn't a migration or a code change to the controller to remove them:

non_va_facility_contact

### PR DESCRIPTION
**Pull request by awong-dev**
fixes #280

### CODE REVIEW

**Comment by awong-dev** [comment]
@kathytpham @emilyville 

This is the pattern for high-level UI tests.  We're still missing (basically the same) block for creating a patient but it's a start.

**Comment by awong-dev** [comment]
@emilyville All yours.

Let me know if you'd prefer to walk through this code together. It has a number of tricks...

File: app/views/patients/_form.html.slim [hunk 1]
```diff
@@ -39,7 +39,7 @@
       h4 VA Status and Travel Eligibility
       .row
         .col-md-6
-          = f.input :assigned_vamc, :label => "Assigned VAMC", collection: Patient.assigned_vamcs.keys, :prompt => "Select one"     
+          = f.input :assigned_vamc, :label => "Assigned VAMC", collection: Patient.collections[:assigned_vamc], :prompt => "Select one"
```
Comment (emilyville) [inline app/views/patients/_form.html.slim]: this better way of doing collections makes the dropdowns only display as text fields. This is sad. Will keep looking into why.

**Comment by emilyville** [comment]
I'm not able to populate all the data from rake db:seed. Thinking either the readme needs to be updated to reflect the new protocol (if that's no longer correct) or you've got an untracked file for the creating the new tables?

Otherwise, it loads locally for me and looks good.

**Comment by awong-dev** [comment]
Right. It's not in rake app:seed_fake. I updated the read-me in a later CL but will try to back-port.

**Comment by emilyville** [comment]
:+1:

**Pull request merged**